composer/trainer/trainer.py

```diff
@@ -1439,7 +1446,11 @@ def _train_loop(self) -> None:
         # surpressing GradScaler warnings as they are always created
         # self._use_grad_scaling() will raise a RuntimeError if grad scaling is not available when it is required
         warnings.filterwarnings(action='ignore', message='torch.cuda.amp.GradScaler')
-        self.state.scaler = ClosureGradScaler() if self._use_closures() else GradScaler()
+        if type(self._device) == DeviceTPU:
+            import torch_xla
+            self.state.scaler = torch_xla.amp.GradScaler
```

Comment:
Can we error if one is using a closure with TPU?

Also, do we need to instantiate it?

```suggestion
            self.state.scaler = torch_xla.amp.GradScaler()
```

Reply:
fixed